src/components/StopPointsEditor/MixedFlexible/MixedFlexibleStopPointEditor.tsx

```diff
@@ -67,22 +69,51 @@ export const MixedFlexibleStopPointEditor = ({
     frontText: frontTextError,
   } = validateStopPoint(stopPoint, isFirst!, isLast!);
 
+  useEffect(() => {
+    // Stop point's mode (nsr or flexible) can change when reordering
```

Comment:
I think we should refrain from using the term "NSR", which is the name of the Norwegian stop place registry. "external" would be better, or maybe "externalQuayRef" or "externalRef"

Reply:
I realize this stems from an existing enum value "StopPlaceMode.NSR", but let's change that as well.